### PR TITLE
Expose GdipFlush in the headers

### PR DESCRIPTION
**Pull request by hughbe**

### CODE REVIEW

File: src/graphics.h [hunk 1]
```diff
@@ -31,6 +31,7 @@ GpStatus WINGDIPAPI GdipBeginContainer (GpGraphics *graphics, GDIPCONST GpRectF*
 GpStatus WINGDIPAPI GdipBeginContainer2 (GpGraphics *graphics, GraphicsContainer* state);
 GpStatus WINGDIPAPI GdipBeginContainerI (GpGraphics *graphics, GDIPCONST GpRect* dstrect, GDIPCONST GpRect *srcrect, GpUnit unit, GraphicsContainer *state);
 GpStatus WINGDIPAPI GdipEndContainer (GpGraphics *graphics, GraphicsContainer state);
+GpStatus WINGDIPAPI GdipFlush(GpGraphics *graphics, GpFlushIntention intention);
```
Comment (akoeplinger) [inline src/graphics.h]: code style: space before opening brace

File: tests/testgraphics.c [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+using namespace Gdiplus;
+using namespace DllExports;
+#endif
+
+static void test_flush()
+{
+    gunichar2 *filePath;
+    GpStatus status;
+    GpImage *image;
+	GpGraphics *graphics;
```
Comment (akoeplinger) [inline tests/testgraphics.c]: code style: mixing tabs/spaces here and below

File: src/graphics.h (hunk 1 above)
Comment (hughbe) [inline src/graphics.h]: Is this just in declarations, or for calling/invoking the function/method as well?

Comment (akoeplinger) [inline src/graphics.h]: Both. It's a general Mono coding style used for most of Mono's libraries.

Comment (hughbe) [inline src/graphics.h]: Thanks - even for parameterless methods?

Comment (akoeplinger) [inline src/graphics.h]: Yes :)

**Comment by hughbe** [comment]
I don't quite understand the CI failure:
> Could not load file or assembly 'System.Runtime, Version=4.0.20.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.

**Comment by akoeplinger** [comment]
No, it's not related. The Linux Travis CI leg is building Mono and that failed for some reason (which is why System.Runtime.dll is missing). I've retriggered that part.

**Pull request merged**